api/v1beta1/designatebackendbind9_types.go

```diff
@@ -63,6 +64,10 @@ type DesignateBackendbind9SpecBase struct {
 	// +kubebuilder:validation:Optional
 	// ServiceAccount - service account name used internally to provide Designate services the default SA name
 	ServiceAccount string `json:"serviceAccount"`
+
+	// +operator-sdk:csv:customresourcedefinitions:type=spec
+	// TLS - Parameters related to the TLS
+	TLS tls.Ca `json:"tls,omitempty"`
```

Comment:
Strictly speaking, I don't think we need this here but it might not hurt in the future. This would be a good thing to run by @johnsom

Reply:
I agree, in the following commit I removed the bind9 TLS parts, I will remove it here as well.